app/overlays/tgl/ipc4_native_drivers_overlay.conf

```diff
@@ -0,0 +1,42 @@
+CONFIG_IPC_MAJOR_4=y
+CONFIG_INTEL_DMIC_NHLT=y
+CONFIG_RIMAGE_SIGNING_SCHEMA="tgl-cavs"
+CONFIG_PCM_CONVERTER_FORMAT_S16LE=y
+CONFIG_PCM_CONVERTER_FORMAT_S24LE=y
+CONFIG_PCM_CONVERTER_FORMAT_S32LE=y
+CONFIG_PCM_CONVERTER_FORMAT_S24_3LE=y
+CONFIG_PCM_CONVERTER_FORMAT_FLOAT=y
+CONFIG_PCM_CONVERTER_FORMAT_S16_C16_AND_S16_C32=y
+CONFIG_PCM_CONVERTER_FORMAT_S16_C32_AND_S32_C32=y
+CONFIG_PCM_CONVERTER_FORMAT_S16_C32_AND_S24_C32=y
+CONFIG_PCM_CONVERTER_FORMAT_S16_C32_AND_S16_C32=y
+
+CONFIG_INTEL_SSP=y
+CONFIG_INTEL_ALH=y
+CONFIG_LP_MEMORY_BANKS=1
+CONFIG_HP_MEMORY_BANKS=22
+
+CONFIG_INTEL_ADSP_TIMER=y
+CONFIG_FORMAT_CONVERT_HIFI3=n
+
+CONFIG_ZEPHYR_NATIVE_DRIVERS=y
+CONFIG_DAI=y
+CONFIG_DAI_INIT_PRIORITY=70
+CONFIG_DAI_INTEL_SSP=y
+CONFIG_DAI_INTEL_DMIC=y
+CONFIG_DAI_INTEL_DMIC_NHLT=y
+CONFIG_DAI_DMIC_HW_IOCLK=38400000
+CONFIG_DAI_DMIC_HAS_OWNERSHIP=y
+CONFIG_DAI_DMIC_HAS_MULTIPLE_LINE_SYNC=y
+
+CONFIG_DMA=y
+CONFIG_DMA_DW_LLI_POOL_SIZE=100
+CONFIG_DMA_DW_FIFO_PARTITION=y
+
+CONFIG_ZEPHYR_LOG=y
+CONFIG_LOG=y
+CONFIG_LOG_MODE_DEFERRED=y
+CONFIG_LOG_BACKEND_ADSP_HDA=y
+CONFIG_LOG_FUNC_NAME_PREFIX_INF=y
+
+CONFIG_HEAP_MEM_POOL_SIZE=8192
```

Comment:
maybe you could add a new line here

Reply:
I'll merge now, we can get this in the next update as I expect there will be some churn here.